Inseok_45/routes/likeRouter.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const likeController = require('../controllers/likeController');
+
+const router = express.Router();
+
+router.put('/', likeController.likePosts);
```

Comment:
```suggestion
const express = require('express');
const likeController = require('../controllers/likeController');

const router = express.Router();

router.put('', likeController.likePosts);
```
엔드포인트 끝에는 `/`가 들어가지 않는것이 REST한 api 엔드포인트 설계 원칙입니다.